libs/encode/json.go

```diff
@@ -19,3 +19,15 @@ func MarshalJSON(msg proto.Message) ([]byte, error) {
 
 	return buf.Bytes(), nil
 }
+
+// MarshalJSONIndent provides an auxiliary function to return Proto3 indented
+// JSON encoded bytes of a message.
+func MarshalJSONIndent(msg proto.Message) ([]byte, error) {
+	jm := &jsonpb.Marshaler{EmitDefaults: false, OrigName: false, Indent: "  "}
```

Comment:
can we make this a global variable to avoid recreating it every time we want to encode something?